apps/admin-portal/src/utils/common-utils.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * This function calculate the number of days since the last
+ * modified date to the current date.
+ *
+ * @param modifiedDate
+ */
+export const handleLastModifiedDate = (modifiedDate: string): string => {
```

Comment:
Shall we show in months as well if the days is more than 30?

Reply:
Shall we use moment js for this, https://www.npmjs.com/package/moment

Reply:
Fixed with commit 184e209. Used MomentJs.